src/vue-pdf-embed.vue

```diff
@@ -13,6 +20,14 @@ pdf.GlobalWorkerOptions.workerPort = new PdfWorker()
 export default {
   name: 'VuePdfEmbed',
   props: {
+    /**
+     * Id of the document to display 
```

Comment:
Technically, this is not a document ID

```suggestion
     * Component identifier (inherited by child SVGs with page number
     * postfixes).
```